vulnerability-disclosure-policy.md

```diff
@@ -67,3 +67,27 @@ Reports should include:
 Please keep your vulnerability reports current by sending us any new information as it becomes available.
 
 We may share your vulnerability reports with [US-CERT](https://www.us-cert.gov/ais), as well as any affected vendors or open source projects.
+
+### Public Disclosure
+
+This public disclosure policy applies to all submissions under the TTS Vulnerability Disclosure Policy, including Duplicates, Out of Scope, and Not Applicable submissions.
```

Comment:
"Duplicates" etc. are not defined in this doc.  "Applies to all submissions" is broad enough to encompass those.